Project-16-Quiz-App/src/components/Answers/index.jsx

```diff
@@ -0,0 +1,53 @@
+import { useRef } from "react";
+import PropTypes from "prop-types";
+import style from "./answers.module.scss";
+
+function Answers({ answers, selectedAnswer, answerState, onSelect }) {
+  const shuffledAnswers = useRef();
+
+  if (!shuffledAnswers.current) {
+    shuffledAnswers.current = [...answers];
+    shuffledAnswers.current.sort(() => Math.random() - 0.5);
+  }
+
+  return (
+    <ul className={style.answers}>
+      {shuffledAnswers.current.map((answer, index) => {
+        const isSelected = selectedAnswer === answer;
+
+        let cssClass = "";
+
+        if (answerState === "answered" && isSelected) {
+          cssClass = style.selected;
+        } else if (
+          (answerState === "correct" || answerState === "wrong") &&
+          isSelected
+        ) {
+          cssClass = style[answerState];
+        }
+
+        return (
+          <li key={answer} className={style.answer}>
```

Comment:
@sknepali  Using the answer as a key in the map function could lead to issues if the answers are not unique. If possible, use a unique id.